### DISCUSSION TITLE
Interpretation / Connect nearby signal parts

### DISCUSSION THREAD

**Discussion by Semnodime** [jopohl/urh#964] [Ideas]
##### Is your feature request related to a problem?
Sometimes the signal is detected as noise when it is only changeing slowly and enters the area near the x-axis and stays there for too long.

------

##### Describe the solution you'd like
Instead of filtering noise only by threshold, instead "connect" two signals if they are very close together. (Introduce a second threshold).

##### Current outcome:
![image](https://user-images.githubusercontent.com/19511852/151687881-fe023ea0-501f-452a-a928-15467365c409.png)

##### Desired outcome: (result of forcing the noise level to zero, picking up the whole signal but also all the garbage)
![image](https://user-images.githubusercontent.com/19511852/151687866-23396954-b9c9-42e1-b640-257baa0c04e0.png)

##### Raw signal waveform:
![image](https://user-images.githubusercontent.com/19511852/151688016-669324f9-c012-4a64-9145-c4b89ef0b5ce.png)

-----

##### Describe alternatives you've considered
Preprocessing the signal in an external program or by hand.

**Comment by jopohl** [comment]
Can you attach a sample signal?
